appium/webdriver/extensions/execute_driver.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from selenium import webdriver
+
+from ..mobilecommand import MobileCommand as Command
+
+
+class ExecuteDriver(webdriver.Remote):
+
+    def execute_driver(self, script, script_type='webdriverio', timeout_ms=None):
+        """Run a set of script against the current session, allowing execution of many commands in one Appium request.
+        Please read http://appium.io/docs/en/commands/session/execute-driver for more details about the acceptable
+        scripts and the output format.
+
+        Args:
+            script (string): The string consisting of the script itself
+            script_type (string): The name of the script type. Defaults to 'webdriverio'.
+            timeout_ms (optional): The number of `ms` Appium should wait for the script to finish before killing it due to timeout_ms.
+
+        Usage:
+            self.driver.execute_driver(script='return [];')
+            self.driver.execute_driver(script='return [];', script_type='webdriverio')
+            self.driver.execute_driver(script='return [];', script_type='webdriverio', timeout_ms=10000)
+
+        Return:
+            ExecuteDriver.Result: The result of the script. It has 'result' and 'logs' keys.
+        """
+
+        class Result(object):
+
+            def __init__(self, response):
+                self.result = response['result']
+                self.logs = response['logs']
+
+        option = {'script': script, 'type': script_type}
+        if timeout_ms is not None:
+            option['timeout'] = timeout_ms
+
+        response = self.execute(Command.EXECUTE_DRIVER, option)['value']
+        return Result(response)
```

Comment:
Below codes can be available.

```
return response['result'], response['logs']
```

Reply:
Yes, but I hope users do not consider the order, [0] is _result_ and [1] is _logs_
thanks